gallery/src/components/NxFilterInput/NxFilterInputPage.tsx

```diff
@@ -25,7 +25,10 @@ const NxFilterInputPage = () =>
   <>
     <GalleryDescriptionTile>
       <NxP>
-        An input to be used for filtering purposes
+        An input to be used for filtering purposes. After a user has text within the input field, a
+        clear filter button appears within the component and is only visible/usable when there is text within
+        the input field. Note: This button is for mouse use only and is therefore not focusable but still
+        hoverable and clickable.
```

Comment:
Maybe add a mention of the clear-on-escape functionality here too

Reply:
Updated here 447daff63725ceb4aa77ad580798f2adac4d1a05. Thanks.